### PR TITLE
Improve Storage Insights by showing error logs

### PR DESCRIPTION
**Pull request by kavyabohra**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

Feature added: 
Providing users an easier way to see their log analytic data so that they can dig into possible errors related to failing transactions. This solution (created by Marcus Heijer) extends the current Failures workbook to display information about failures in terms of logs. The following scenarios are handled:

The user has enabled diagnostic settings and is able to view logs. 
show logs

The user has enabled diagnostic settings but no logs available for the selected time range. 
no logs to show

The user has not enabled diagnostic settings and logs are unavailable. 
no logs

Descriptions for each Individual Storage workbook updated: 
- Overview: 
overview
- Failures:
failures 1
- Performance: 
performance
- Availability: 
avail
- Capacity
cap

nit fixes: 
- Turning on "Open in Metrics Explorer" functionality for all charts. 
- Updated "Sum" to "Count | Sum" for number of transactions on Overview to achieve consistency in both Aggregation type value and unit on all metrics
- Ingress renamed to Bandwidth in Overview
BANDWIDTH CHANGES
- Error table color scheme changed from blue to red. 
TABLE CHANGES

Contacts: Francis Yu/Marcus Heijer (PM), Kavya Bohra (dev)
Deploying own gallery at: https://portal.azure.com/?feature.workbookGalleryRedirect=https://localworkbooks.blob.core.windows.net/azure-monitor-workbook-templates/package

### CODE REVIEW

**Comment by gardnerjr** [comment]
my only nit would be do you want that text *before* all the things?  is it more important than the data itself?  after a user has seen it the first time and knows that this is metrics data, it is always there, at the top, taking space before the important thing (the numbers).  should all that text be after the numbers?

File: Workbooks/Individual Storage/Failures/Failures.workbook [hunk 1]
```diff
@@ -37,6 +37,90 @@
...
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "26332d26-265f-4cef-8b21-8ee048ffd8a8",
+            "version": "KqlParameterItem/1.0",
+            "name": "Logs",
+            "type": 1,
+            "isRequired": true,
+            "query": "(StorageBlobLogs | take 1)\r\n| union (StorageFileLogs | take 1)\r\n| union (StorageQueueLogs | take 1)\r\n| union (StorageTableLogs | take 1)\r\n| summarize c = count()\r\n| project isMissing=iff(c > 0, 0, 1)\r\n| extend isMissing = tostring(isMissing)",
```
Comment (gardnerjr) [inline Workbooks/Individual Storage/Failures/Failures.workbook:51]: are all these tables guaranteed to exist?  or can this throw tablenotfound/etc errors?  do you need to protect for that with isfuzzy?  see https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/DataSources/LogsBestPracticesAndHints.md#protect-against-a-table-that-might-not-exist

Comment (kavyabohra) [reply Workbooks/Individual Storage/Failures/Failures.workbook:51]: > are all these tables guaranteed to exist? or can this throw tablenotfound/etc errors? do you need to protect for that with isfuzzy? see https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/DataSources/LogsBestPracticesAndHints.md#protect-against-a-table-that-might-not-exist

Yes, the tables are guaranteed to exist; Marcus Heijer verified this.

**Comment by kavyabohra** [comment]
> my only nit would be do you want that text _before_ all the things? is it more important than the data itself? after a user has seen it the first time and knows that this is metrics data, it is always there, at the top, taking space before the important thing (the numbers). should all that text be after the numbers?

I reduced the extra margins on the numbered list component to make it a little more compact. We would like to have this data present, especially on the Failures tab as it helps guide the user towards the newly added logs feature. I have attached a screenshot of what the updated heading looks like below: 
image

**Review by gardnerjr — APPROVED** [review]

**Pull request merged**